src/napari_cryoet_data_portal/_open_widget.py

```diff
@@ -135,20 +135,18 @@ def _loadTomogram(
         image_scale = image_layer[1]["scale"]
         yield _handle_image_at_resolution(image_layer, resolution)
 
-        # Looking up tomogram.tomogram_voxel_spacing.annotations triggers a query
-        # using the client from where the tomogram was found.
         # A single client is not thread safe, so we need a new instance for each query.
         client = Client(self._uri)
-        annotations = Annotation.find(
+        anno_files = AnnotationFile.find(
             client,
             [
-                Annotation.tomogram_voxel_spacing_id
+                AnnotationFile.tomogram_voxel_spacing_id
```

Comment:
```suggestion
                AnnotationFile.tomogram_voxel_spacing_id == tomogram.tomogram_voxel_spacing_id,
                AnnotationFile.alignment_id == tomogram.alignment_id
```
We should make sure alignments *and* voxel spacings match

Reply:
Ah, thanks for the catch!